src/Transition/CreateNewIDFUsingRulesV8_6_0.f90

```diff
@@ -728,6 +727,36 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
                   OutArgs(1)='*'
                   nodiff=.false.
                 ENDIF
+
+                ! For Output:Variable that reference a specific Daylighting:Controls object need update to the new reference point name
+                IF (InArgs(1) .NE. '*') THEN
+                  isDElightOutVar = .FALSE. ! first assume that it is not a DElight related variable
+                  IF (SameString(InArgs(2)(1:27),'Daylighting Reference Point')) THEN
```

Comment:
Oh, I see now, the outputs are currently dependent on the method, the numbered approach for SplitFlux and the reference point name approach for DElight.  I hadn't understood that before.   How about setting up both types for both methods?